src/api.js

```diff
@@ -1,15 +1,22 @@
 import express from "express";
+import { postReview } from "./addReviewServer.js";
 import getMovieRating from "./getMovieRating.js";
 import { loadMovie } from "./movies.js";
 import { getReviewsForMovieWithId } from "./reviewList.js";
 
-
 const apiRouter = express.Router();
 
 apiRouter.get("/movies/:id", (req, res) => {
-	res.send(`movie with id of ${req.params.id}`);
+  res.send(`movie with id of ${req.params.id}`);
 });
 
+apiRouter.post(
+  "/movies/:movieId/reviews",
+ express.json(),
+  async (req, res) => {
+    postReview(req, res);
```

Comment:
Like @davidlidebrandt already suggested in his comment, you don't have to wrap this function in an unnamed, anonymous function the way you're doing here.

Instead of this:

```js
apiRouter.post(
  "/movies/:movieId/reviews",
  express.json(),
  async (req, res) => { // The anonymous function starts here…
    postReview(req, res);
  } // …and ends here
);
```

You could just do this:

```js
apiRouter.post(
  "/movies/:movieId/reviews",
  express.json(),
  postReview
);
```